eth-account-snapshot/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict'
 const EthAccount = require('ethereumjs-account')
-const cidFromHash = require('eth-hash-to-cid')
+const cidFromHash = require('../util/cidFromHash')
```

Comment:
please use the convention of `-` instead of Camel Case for filenames.

Reply:
I did it in favour of consistency. All files in `util` are camel case.

Reply:
favor consistency across modules.

Reply:
Generally yes, but in this case favour local consistency. I've opened an issue for global consistency: https://github.com/ipld/js-ipld-ethereum/issues/20